core/src/mindustry/net/Administration.java

```diff
@@ -304,6 +304,32 @@ public Seq<PlayerInfo> getBanned(){
     public Seq<String> getBannedIPs(){
         return bannedIPs;
     }
+    
+    /**
+     * Returns list of all players which are kicked
+     */
+    public Seq<PlayerInfo> getKicked(){
+        Seq<PlayerInfo> result = new Seq<>();
+        for(PlayerInfo info : playerInfo.values()){
+            if(info.lastKicked > Time.millis()){
+                result.add(info);
+            }
+        }
+        return result;
+    }
+    
+    /**
+     * Returns all kicked IPs. This does not include the IPs of ID-kicked players.
+     */
+    public ObjectMap<String, Long> getKickedIPs(){
```

Comment:
There's already `kickedIPs` (Administration.java, line 25) for that.

Reply:
Ok, i will try